### PR TITLE
Footer team

### PR DESCRIPTION
**Pull request by julianecasier**

### CODE REVIEW

**Review by ghost — APPROVED** [review]

File: frontend/src/components/Footer/index.jsx [hunk 1]
```diff
@@ -1,19 +1,35 @@
...
-        <p>CINERAMA</p>
-        <p> Avenue des Peupliers 59000 Lille</p>
-        <p>59000 Lille</p>
-      </Modal>
-      <Modal button="Team">
-        <h2>Hello, this is own team ! </h2>
-        <p> Describe own team project</p>
-      </Modal>
+      <div className="entry">
+        <Modal button="Show Adress">
```
Comment (LuigiMenez) [inline frontend/src/components/Footer/index.jsx:10]: Attention a name props

File: frontend/src/components/Footer/index.jsx [hunk 2]
```diff
@@ -1,19 +1,35 @@
...
+          <h2>Own adress </h2>
+          <p>CINERAMA</p>
+          <p> Avenue des Peupliers 59000 Lille</p>
+          <p>59000 Lille</p>
+        </Modal>
+      </div>
+      <div className="entry">
+        <Modal button="Team">
```
Comment (LuigiMenez) [inline frontend/src/components/Footer/index.jsx:18]: attention a name props

**Review by LuigiMenez — APPROVED** [review]
Be careful with the name you give to your props

**Review by LoicBrassart — APPROVED** [review]
Saw non Evil

**Pull request merged**